http/1.9/modules/ROOT/pages/http-connector-config-acb.adoc

```diff
@@ -0,0 +1,252 @@
+= Using Anypoint Code Builder to Configure HTTP Connector
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="individual-connector-config"]
+
+=== Configuring Attributes for the HTTP Listener
+
+The Anypoint Connector for HTTP (HTTP Connector) *Listener* component enables you to set up an HTTP server and trigger flows when receiving HTTP requests.
+
+To configure the HTTP *Listener* attributes in the UI:
+
+. Click the *Listener* component on the canvas.
++
+The configuration UI opens.
+. In the *General* tab, enter the relative path for the path set in the HTTP configuration element in *Path*. See xref:http-listener-ref.adoc#configure-paths[Configure Paths]. 
+. Select the *Advanced* tab to configure the following attributes:
+* *Allowed methods*
++
+Comma-separated list of allowed HTTP methods by this listener. To allow all methods, do not define this attribute.
++
+* *Response streaming mode*
++
+Defines whether or not to use streaming when the response is sent.
++
+* *Output MIME type*
++
+Defines the format of the response.
++
+* *Parameters* 
++
+Add additional parameters for the *Listener*.
++
+* *Output encoding*
++
+Select the encoding to transform the data to.
++
+* *Streaming Strategy*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:connectors::streaming-strategies.adoc[]
++
+* *Primary node only*
++
+Check this box to execute the *Listener* trigger on the primary node only when running in a cluster.
+* *Redelivery policy*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#RedeliveryPolicy[Redelivery Policy].
+* *Connection*
+** *Reconnection strategy*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#reconnect[Reconnect] and xref:http-documentation.adoc#reconnect[Reconnect Forever].
+* *Reponse*
++
+See xref:http-documentation.adoc#HttpResponseAttributes[HTTP Respoonse Attributes]
++
+* *Error Response*
++
+An HTTP error response returns a status code for the error. 
```

Comment:
Not parallel with the other bullets, but neither are the first few, so disregard.